src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, DATE_FORMATTER);
+        LocalDate localDateTo = LocalDate.parse(dateTo, DATE_FORMATTER);
+
+        StringBuilder report = new StringBuilder();
+        report.append("Report for period ").append(dateFrom)
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder()
                                                 .append("Report for period ")
                                                 .append(dateFrom)
```
not a mistake but FYI